api/src/utils/jsonld/index.ts

```diff
@@ -9,7 +7,7 @@ export class JsonldGenerator {
 		Service: 'https://schema.org/'
 	};
 
-	jsonldUserData(userType: string, data: Person | Organization | Service | Device | Product) {
+	jsonldUserData(userType: string, data: any) {
```

Comment:
Do you expect a payload which can be different from the defined types of `Person | Organization | Service | Device | Product` ?

Reply:
yes since companies using the api shall not be restricted to only these types we can imagine but have the possibility to define their own types do you think
```
Person | Organization | Service | Device | Product | any
```

would be better?

Reply:
> would be better?

not really, as it would still allow to pass an object which is to 80% similar to `Device` or `Person`, but still different

Reply:
let's leave it as `any`, it's not a show-stopper